roles/custom/matrix-synapse-usage-exporter/defaults/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+
+# Synapse Usage Exporter
+# Project source code URL: https://github.com/loelkes/synapse-usage-exporter/tree/main
```

Comment:
```suggestion
# Project source code URL: https://github.com/loelkes/synapse-usage-exporter
```

I wonder if `/tree/main` makes our feed generator choke on this URL. Maybe it doesn't, but.. it's better to use the canonical repository URL just in case.